haiku-games/bong/bong-1.4~git.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="BeOS version of Pong"
+DESCRIPTION="Bong is a FreeWare game for BeOS,based on the old arcade classic Pong"
+HOMEPAGE="https://github.com/HaikuArchives/Bong"
+COPYRIGHT="1997 by Gertjan van Ratingen"
+LICENSE="bong"
+REVISION="1"
+srcGitRev=906d822b79bdae9328fe743dffe2871f96a30e0e
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+SOURCE_FILENAME="bong-1.4~$srcGitRev.tar.gz"
+SOURCE_DIR="Bong-$srcGitRev"
+
+PATCHES="bong-$portVersion.patchset"
+PATCH()
```

Comment:
Move PATCH() to before BUILD() as in the [guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines)

Also, there's no need for a space before `PATCHES`